library/mlops/src/iteration.py

```diff
@@ -17,6 +21,9 @@ def __init__(self, iteration_name: str, project_id: str = None, experiment_id: s
         self.path_to_model: str = ""
         self.parameters: dict = {}
         self.metrics: dict = {}
+        self.hasDataset: bool = False
+        self.dataset_id: str = None
```

Comment:
I'd store dataset as only `self.dataset_id: str = None`, we don't need to store `dataset_name` and `version`, since they are autmatically updated when adding dataset using ID on back-end.

Reply:
corrected